src/document/DocumentAnnotator.ts

```diff
@@ -57,6 +58,18 @@ export default class DocumentAnnotator extends BaseAnnotator {
         return this.annotatedEl ? Array.from(this.annotatedEl.querySelectorAll('.page')) : [];
     }
 
+    toggleAnnotationMode(mode: Mode): void {
+        super.toggleAnnotationMode(mode);
+
+        if (!this.annotatedEl) {
+            return;
+        }
+
+        if (mode === Mode.HIGHLIGHT) {
+            this.annotatedEl.classList.toggle('is-highlighting');
```

Comment:
It looks like this will be left stale if the mode is set to something other than `Mode.HIGHLIGHT`?